tests/integration/conftest.py

```diff
@@ -24,6 +24,42 @@ def single_passing_test_for_asc(_zigzag_runner_factory, asc_config_file, asc_glo
     return zz_runner
 
 
+@pytest.fixture(scope='session')
+def single_passing_test_with_custom_build_url(_zigzag_runner_factory,
+                                              custom_build_url_config_file,
+                                              custom_config_global_props):
+    """ZigZag CLI runner with custom configuration.
+
+    Returns:
+        ZigZagRunner
+    """
+
+    zz_runner = _zigzag_runner_factory('single_passing_asc.xml',
+                                       custom_build_url_config_file,
+                                       custom_config_global_props)
+    zz_runner.add_test_case('passed')
+
+    return zz_runner
+
+
+@pytest.fixture(scope='session')
+def single_passing_test_with_custom_mod_hierarchy(_zigzag_runner_factory,
+                                                  custom_hierarchy_config_file,
+                                                  custom_config_global_props):
+    """ZigZag CLI runner with custom configuration.
```

Comment:
Description is vague.

Reply:
addressing in fb1987cbd8f4107ef7aba96c807d6cadfa7074dd